### DISCUSSION TITLE
How to change filetype rules of `synlink`?

### DISCUSSION THREAD

**Discussion by Jaehaks** [sxyazi/yazi#1586] [1. Q&A] [answered]
I found that symbolic link of result of file() is `inode/symlink` in windows. 
And I added like this in theme.toml but it doesn't work.
```toml
[filetype]
{mime = "inode/symlink", fg="cyan"}
```
other filetypes works properly. but only symlink is dealed with fallback property
What do i missed?

**Comment by sxyazi** [comment, marked as answer]
Try `is = "link"`:

```sh
{ name = "*", is = "link", fg = "cyan" }
```

Doc https://yazi-rs.github.io/docs/configuration/theme#filetype

**Reply by Jaehaks** [reply]
@sxyazi 

Thanks:)
